### PR TITLE
To fix custom spell effect

### PR DESCRIPTION
**Pull request by 5840174**

### CODE REVIEW

File: src/server/game/Server/Protocol/Opcodes.cpp [hunk 1]
```diff
@@ -133,1440 +133,1192 @@ void OpcodeTable::Initialize()
...
+    DEFINE_SERVER_OPCODE_HANDLER(SMSG_DISPLAY_PLAYER_CHOICE,                                   STATUS_NEVER);     // 5.4.8 18414
+    DEFINE_SERVER_OPCODE_HANDLER(SMSG_DISPLAY_PROMOTION,                                       STATUS_NEVER);     // 5.4.8 18414
+    DEFINE_SERVER_OPCODE_HANDLER(SMSG_DISPLAY_TOAST,                                           STATUS_NEVER);     // 5.4.8 18414
+    DEFINE_SERVER_OPCODE_HANDLER(SMSG_DUEL_COMPLETE,                                           STATUS_NEVER);     // 5.4.8 18414
+    DEFINE_SERVER_OPCODE_HANDLER(SMSG_DUEL_COUNTDOWN,                                          STATUS_NEVER);     // 5.4.8 18414
+    DEFINE_SERVER_OPCODE_HANDLER(SMSG_DUEL_INBOUNDS,                                           STATUS_NEVER);     // 5.4.8 18414
+    DEFINE_SERVER_OPCODE_HANDLER(SMSG_DUEL_OUTOFBOUNDS,                                        STATUS_NEVER);     // 5.4.8 18414
+    DEFINE_SERVER_OPCODE_HANDLER(SMSG_DUEL_REQUESTED,                                          STATUS_NEVER);     // 5.4.8 18414
+    DEFINE_SERVER_OPCODE_HANDLER(SMSG_DUEL_WINNER,
```
Comment (leelf00) [inline src/server/game/Server/Protocol/Opcodes.cpp:1567]: Do not remove these comments

**Comment by leelf00** [comment]
How to test this PR?

**Comment by 5840174** [comment]
// x - skip<uint32>, X - skip<uint8>, s - char*, S - char*, f - float, i - uint32, b - uint8, d - index (not included)
// n - index (included), l - bool, p - field present in sql dbc, a - field absent in sql dbc

我将其中的apppap修改为papppa
因为spelleffect.dbc的EffectItemType和EffectPointsPerCombo字段并不存在于数据库中 
但在CustomSpellEffectEntryfmt的定义中它们的位置出现了错误
猜测是早期开发者的疏漏导致字段顺序错误

**Comment by 5840174** [comment]
至于怎么测试你自己想办法吧 
我是偶然发现这个错误的

**Comment by 5840174** [comment]
我是偶然发现spelleffect_dbc表的EffectMiscValueB字段没有载入数据

**Comment by leelf00** [comment]
spell_dbc table is not loaded in core. This is a missing part of core function.

**Comment by 5840174** [comment]
The core function is not missing the loading of the spell_debc table... it's just an error in the field order...
Okay, you can use“. cast 24308” to check the error log of the console
the error log will indicate that the value of type is 0.
Now you can apply my changes and test it out

**Comment by 5840174** [comment]
Haha, forget it

**Pull request closed without merging**